internal/fs/inode/file_test.go

```diff
@@ -682,3 +682,36 @@ func (t *FileTest) SetMtime_SourceObjectMetaGenerationChanged() {
 	ExpectEq(newObj.Generation, o.Generation)
 	ExpectEq(newObj.MetaGeneration, o.MetaGeneration)
 }
+
+func (t *FileTest) ContentEncodingGzip() {
+	// Set up an explicit mtime on the backing object and re-create the inode.
+	if t.backingObj.Metadata == nil {
+		t.backingObj.ContentEncoding = "gzip"
+	}
+
+	t.createInode()
+
+	AssertEq(t.in.Source().ContentEncoding, "gzip")
+}
+
+func (t *FileTest) ContentEncodingNone() {
+	// Set up an explicit mtime on the backing object and re-create the inode.
+	if t.backingObj.Metadata == nil {
+		t.backingObj.ContentEncoding = ""
+	}
+
+	t.createInode()
+
+	AssertEq(t.in.Source().ContentEncoding, "")
```

Comment:
Please correct here and all other places.

Reply:
Done